docs/build_process.rst

```diff
@@ -359,3 +378,18 @@ Note:
 - where the worker build log line had no timestamp (perhaps the log
   line had an embedded newline, or was logged outside the adapter
   using a different format), the line was left alone
+
+
+Autorebuilds
+------------
+
+OSBS’s autorebuild feature automatically starts new builds of layered images
+whenever the base parent image changes. This is particularly useful for image
+owners that maintain a large hierarchy of images, which would otherwise require
+manually starting each image build in the correct order.  Instead, image owners
+can start a build for the topmost ancestor which upon completion triggers the
+next level of layered images, and so on.
+
+Builds may opt in to autorebuilds with an
+:ref:`autorebuild entry in the dist-git configuration. <container.yaml-autorebuild>`
```

Comment:
@twaugh just a quick question should we update our chainrebuilds test to use container.yaml instead of .osbs-repo-config ??

Reply:
No. The "whether" configuration comes from .osbs-repo-config, but the "how" comes from container.yaml.

Reply:
I see thanks